src/gz/gz.c

```diff
@@ -1230,12 +1232,20 @@ ENTRY void _start()
     menu_add_static(&menu_misc, 2, 11, "language", 0xFFFFFF);
     menu_add_option(&menu_misc, 11, 11, "japanese\0""english\0",
                     byte_option_proc, &language_option_data,  0);
-    menu_add_button(&menu_misc, 2, 12, "pause / unpause", pause_proc,
+    static int8_t target_options[] = {0x00, 0x01};
+    static struct byte_option target_option_data =
+    {
+      &z64_file.target, target_options, 2,
+    };
+    menu_add_static(&menu_misc, 2, 12, "targetting", 0xFFFFFF);
```

Comment:
this option should preferrably use the more common spelling `targeting`, or `z targeting`, as it appears in the option menu of the game itself